### ISSUE TITLE
Add text to the picture questions 

### ISSUE THREAD

**Issue by leeoades** [codemortals/game-platform#76]
The picture round displays an image only. Provide the facility to add the text question.
Eg the text could say "Who is this person?" or "Which sata cable is this?" (!)

**Comment by eddiejaoude** [comment]
Yep 👍 . We have markdown in the normal question, so I personally don't think we need a picture type anymore - I think @andrew-cunliffe feels differently though?
